src/docker/compose.local.yaml

```diff
@@ -1,51 +1,63 @@
 # yamllint disable rule:empty-lines rule:key-ordering rule:brackets
+# cspell:word localnet
 ---
 services:
   aptos-node:
     network_mode: 'host'
-    build:
-      context: '../../'
-      dockerfile: 'src/docker/aptos-node/Dockerfile'
+    image: 'econialabs/aptos-cli:4.1.0'
+    profiles: ["test-harness"]
     ports:
     - '50051:50051'
     - '8070:8070'
     - '8090:8090'
     - '8080:8080'
     - '8081:8081'
     healthcheck:
```

Comment:
You don't need this. It actually gives me a warning when running `docker compose up`. When `network_mode` is set to `host`, this is ignored.

Reply:
Thanks for noting, was wondering why that warning kept coming up, I'll remove it